FakeServer/Common/ObjectHelper.cs

```diff
@@ -186,6 +211,26 @@ public static dynamic GetValueAsCorrectType(string value)
             return value;
         }
 
+        /// <summary>
+        /// Convert a value to correct type
+        /// </summary>
+        /// <param name="value">the input value</param>
+        /// <returns>value as a boolean, an integer, a double, a DateTime or as a string</returns>
+        public static dynamic GetValueAsCorrectType(string value)
```

Comment:
One line methods should be written (haven't always followed that myself either 😄)
```cs
public static dynamic GetValueAsCorrectType(string value) => GetValueAsCorrectType(value, _convertValueFuncs);
```